modules/videoio/src/cap_ffmpeg_api.hpp

```diff
@@ -30,7 +30,9 @@ enum
     CV_FFMPEG_CAP_PROP_CODEC_PIXEL_FORMAT=46,
     CV_FFMPEG_CAP_PROP_BITRATE=47,
     CV_FFMPEG_CAP_PROP_ORIENTATION_META=48,
-    CV_FFMPEG_CAP_PROP_ORIENTATION_AUTO=49
+    CV_FFMPEG_CAP_PROP_ORIENTATION_AUTO=49,
+    CV_FFMPEG_CAP_PROP_OPEN_TIMEOUT_MSEC=50,
+    CV_FFMPEG_CAP_PROP_READ_TIMEOUT_MSEC=51
```

Comment:
Need to restore defined enum values in videoio.hpp. It is a public API header, from which users know how to use this feature.

Also please use 53, 54 as values to align with master (for both "enums").

Reply:
ok will do now

Reply:
Done